QUEST_LV_0100.tsv

```diff
@@ -8119,12 +8119,12 @@ QUEST_LV_0100_20150918_008114	The Writings on the Wall
 QUEST_LV_0100_20150918_008115	Which factors are decisive for determining the outcome of battle?{nl}If you just continue to attack thoughtlessly, then you may get attacked instead.
 QUEST_LV_0100_20150918_008116	Golems in this situation...{nl}You remember what I taught you right?
 QUEST_LV_0100_20150918_008117	$You are doing great.{nl}As a Paladin Master, I must pay respect to your achievements.
-QUEST_LV_0100_20150918_008118	Thank you for passing Laimonas' regards.{nl}You should enter Klaipeda.{memo Uska is a knight, keep the Sir} Sir Uska is waiting for the Revelators.
+QUEST_LV_0100_20150918_008118	Thank you for completing Laimonas' favor.{nl}You may enter Klaipeda.{memo Uska is a knight, keep the Sir} Sir Uska is waiting for the Revelators.
 QUEST_LV_0100_20150918_008119	$There might be more of these Large Kepas roaming in the area nearby.{nl}Could you take a look for me, please?{nl}
 QUEST_LV_0100_20150918_008120	$It was just as I thought, right? It was this instinct that kept me alive during Medzio Diena.
 QUEST_LV_0100_20150918_008121	$It's been too long, my men should've come back, but I haven't heard anything from them yet.{nl}I know it's been four years since Medzio Diena, but my men have been getting too relaxed recently.
 QUEST_LV_0100_20150918_008122	
-QUEST_LV_0100_20150918_008123	The Goddesses disappearing, things like golems appearing... {nl}Is there really any hope?
+QUEST_LV_0100_20150918_008123	With the Goddesses disappearing, and monsters like the golems appearing... {nl}Is there really any hope?
```

Comment:
golems -> Golem

(It reads better when you say: "and monsters like the Golem appearing.." 

also, you encounter one.